src/plugins/intel_gpu/src/graph/impls/common/register.cpp

```diff
@@ -15,7 +15,6 @@ void register_implementations() {
     REGISTER_COMMON(data);
     REGISTER_COMMON(input_layout);
     REGISTER_COMMON(loop);
-    REGISTER_COMMON(prior_box);
```

Comment:
Yes, please keep backward compatibility to PriorBox-1.

Reply:
New PriorBox-8 PR where comments from this one were addressed - [https://github.com/openvinotoolkit/openvino/pull/12995](https://github.com/openvinotoolkit/openvino/pull/12995) . I had to create new one because cannot push to the branch related to this one. 
Some notes regarding Common implementation and other issues will follow in email.